src/groovy/au/org/emii/portal/HostVerifier.groovy

```diff
@@ -12,6 +12,9 @@ class HostVerifier {
     def allowedHosts = null
 
     def allowedHost(address) {
+        if (grailsApplication.config.allowAnyHost) {
+            return true
+        }
 
         initializeAllowedHostsIfNeeded()
 
```

Comment:
I think this bypass is best at the beginning. It's allowing any host, also empty host. Everything is allowed. This is practically short-circuiting the function.